packages/story-editor/src/app/currentUser/currentUserProvider.tsx

```diff
@@ -24,15 +24,10 @@ import type { PropsWithChildren } from 'react';
  */
 import { useAPI } from '../api';
 import type { User } from '../../types/configProvider';
-import type {
-  CurrentUserState,
-  UpdateCurrentUserProps,
-} from '../../types/currentUserProvider';
+import type { CurrentUserState } from '../../types/currentUserProvider';
 import Context from './context';
 
-function CurrentUserProvider({
-  children,
-}: PropsWithChildren<Record<string, never>>) {
+function CurrentUserProvider({ children }: PropsWithChildren<unknown>) {
```

Comment:
Why not `Record<string, never>` as it was before?

Reply:
It caused errors when actually using it in a typed component. This is the proper way to type it (and the default in React 18 where the type is updated to `PropsWithChildren<T = unknown>`).